reference/lexical_analysis.po

```diff
@@ -108,7 +108,6 @@ msgid "Comments"
 msgstr "Commentaires"
 
 #: ../Doc/reference/lexical_analysis.rst:70
-#, fuzzy
```

Comment:
La version anglaise a supprimé la fin de la dernière phrase : « they are not tokens ». Sur bpo, la raison indiquée est que cela peut induire en erreur le lecteur. Je ne vois pas comment, mais il me paraît dangereux de nous écarter de la version originale.
Je supprimerais donc « ce ne sont pas des lexèmes. »